src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+
+* {
+  line-height: 1;
+  box-sizing: border-box;
```

Comment:
Only `box-sizing` property allowed for `*` selector. Move last styles to another class